tests/daos/test_dao.py

```diff
@@ -117,3 +144,42 @@ def test_add_test_result(self):
                 TestResult(True, "0.18.2", TestType.DEV_COMPATIBLE),
             ],
         )
+
+    def test_update_badges(self):
```

Comment:
Can we move tests for badge to `test_manager` file or other appropriate place 😄 
It should not live in database tests.

Reply:
Nevermind, I did it :)